src/garage/torch/algos/ppo.py

```diff
@@ -75,16 +80,17 @@ def __init__(self,
                          policy_ent_coeff=policy_ent_coeff,
                          use_softplus_entropy=use_softplus_entropy,
                          stop_entropy_gradient=stop_entropy_gradient,
-                         entropy_method=entropy_method)
+                         entropy_method=entropy_method,
+                         minibatch_size=minibatch_size,
+                         max_optimization_epochs=max_optimization_epochs)
 
         self._lr_clip_range = lr_clip_range
 
-    def _compute_objective(self, advantages, valids, obs, actions, rewards):
+    def _compute_objective(self, advantages, obs, actions, rewards):
         """Compute objective using surrogate value and clipped surrogate value.
 
         Args:
             advantages (torch.Tensor): Expected rewards over the actions
-            valids (list[int]): length of the valid values for each path
             obs (torch.Tensor): Observation from the environment.
             actions (torch.Tensor): Predicted action.
```

Comment:
please correct the definitions of these parameters. i cover in a comment below how they are not correct

Reply:
fixed